sasdata/temp_ascii_reader.py

```diff
@@ -194,9 +189,18 @@ def load_data(params: AsciiReaderParams) -> list[SasData]:
     loaded_data: list[SasData] = []
     for filename in params.filenames:
         quantities = load_quantities(params, filename)
-        metadata = metadata_to_data_backing(
+        raw_metadata = import_metadata(
             params.metadata.all_file_metadata(path.basename(filename))
         )
+        metadata = Metadata(
+            title=None,
+            run=[],
+            definition=None,
+            sample=None,
+            instrument=None,
+            process=None,
+            raw=raw_metadata,
+        )
```

Comment:
Given the raw metadata, is it not possible to determine any of the other fields at this point?